### PR TITLE
pulp_container_content: Support state=read for container-push repos

### PR DESCRIPTION
**Pull request by markgoddard**
This allows us to check whether a tag exists in a container-push repository.

### CODE REVIEW

**Review by jovial — APPROVED** [review]

**Review by jovial — DISMISSED** [review]
Failure looks unrelated to me. Happy for it to be fixed in another PR.

File: tests/pulp-in-one.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ set -o pipefail
 
 mkdir -p settings
 
-PULP_TAG=${PULP_TAG:-"3.21"}
+PULP_TAG=${PULP_TAG:-"3.45"}
```
Comment (jovial) [inline tests/pulp-in-one.sh:11]: Hmm, do we need to update this now: https://github.com/stackhpc/ansible-collection-pulp/blob/master/.github/workflows/pull_request.yml#L19-L20?

**Review by jovial — APPROVED** [review]
Required checks needs updating for new names

**Comment by markgoddard** [comment]
> Required checks needs updating for new names

:+1: will update

**Pull request merged**